pkgs/development/python-modules/astropy/default.nix

```diff
@@ -83,6 +92,8 @@ buildPythonPackage rec {
     "test_sidereal_lon_independent"
     "test_timedelta_full_precision_arithmetic"
     "test_datetime_to_timedelta"
+
+    "test_datetime_difference_agrees_with_timedelta_no_hypothesis"
```

Comment:
How is this related to the malware? The source code for the test doesn't seem to be related to the HTML template:

https://github.com/astropy/astropy/blob/v6.0.1/astropy/time/tests/test_precision.py#L759

The package seem to build fine as of today:

https://hydra.nixos.org/build/263062097

Reply:
It's not but the test failed multiple times with it on my machine. It's also quite close to some that were disabled recently for the same reason.